### ISSUE TITLE
[Need to control variables] Vulkan cause black screen in macOS 15

### ISSUE THREAD

**Issue by Halo-Michael** [hrydgard/ppsspp#19594]
### Game or games this happens in

-

### What area of the game / PPSSPP

I have just built a hackintosh machine with 12900F+AMD 6750GRE(important). PPSSPP shows black screen when switch graphics backend to vulkan. But I can still tap anything like normal, games will work and play sounds normal with black screen. Unfortunately I have checked old versions from release pages already and it doesn't works too. I have checked the log of moltenVK framework, and it just shows everything find. By the way, vkcubepp works well. And I used have another hackintosh machine with 9600k IGPU UHD630, I remember vulkan do works. So I don't know if it's cause by my graphic card 6750GRE (which native Mac doesn't have, driven by NootRX.kext) or PPSSPP itself. I think control variables is in need later. But I just push this issues here first in case if any of you know the reason.
![截屏2024-11-06 04 56 07](https://github.com/user-attachments/assets/709d29db-b97b-412c-a359-dc338363797d)
![截屏2024-11-06 04 56 22](https://github.com/user-attachments/assets/df397a26-d381-4324-b6bc-0c46f50d83cd)
![截屏2024-11-06 04 56 44](https://github.com/user-at
![截屏2024-11-06 05 08 33](https://github.com/user-attachments/assets/31a0f2aa-fd43-4d6c-a322-cba1e4fe4590)
tachments/
[PPSSPPGold.txt](https://github.com/user-attachments/files/17638580/PPSSPPGold.txt)
assets/e395b8ab-285f-4abe-9e5a-15758dc6f7f4)

### What should happen

Vulkan works

### Logs

[PPSSPPGold.txt](https://github.com/user-attachments/files/17638587/PPSSPPGold.txt)

### Platform

macOS

### Mobile device model or graphics card (GPU)

AMD 6750GRE

### PPSSPP version affected

1.18.1-40-gc9c543

### Last working version

-

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
No clue what the problem exactly is, it's well known though that PPSSPP fails this way on many Macs with AMD. Fortunately you can now easily get out of this state by switching to OpenGL in the menu at the top of the screen, but it's definitely not ideal.

I don't know how to debug this though without buying an AMD Mac, and the amount of users affected is really small these days so haven't prioritized it.

**Comment by Halo-Michael** [comment]
Yeah I tried my 9600K IGPU hackintosh machine, and PPSSPP works fine with vulkan backend. So you means PPSSPP is just have no support with AMD dGPUs in macOS right now?

> No clue what the problem exactly is, it's well known though that PPSSPP fails this way on many Macs with AMD. Fortunately you can now easily get out of this state by switching to OpenGL in the menu at the top of the screen, but it's definitely not ideal.
> 
> I don't know how to debug this though without buying an AMD Mac, and the amount of users affected is really small these days so haven't prioritized it.

**Comment by hrydgard** [comment]
Apparently so. I don't know exactly what works because, like I said, I don't have the hardware to test.

**Comment by anr2me** [comment]
hackintosh works on a PC instead of a real Mac machine isn't?

**Comment by hrydgard** [comment]
I suppose I could try to install hackintosh on my secondary workstation that uses AMD, but I don't really want to mess too much with that machine, and the AMD card might be too modern to work with it anyway, not sure (RX460 I think)?

**Comment by Halo-Michael** [comment]
Well, AMD cards are always welcome (except RX7xxx family).

> I suppose I could try to install hackintosh on my secondary workstation that uses AMD, but I don't really want to mess too much with that machine, and the AMD card might be too modern to work with it anyway, not sure (RX460 I think)?
